content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,169 @@
+:guilabel:`New` button in the top-left corner.
+
+The :guilabel:`Subject / Application` field is populated with the job position, by default. Certain
+fields on the applicant card may be pre-populated, depending on how the job position is configured.
+Typically, the :guilabel:`Job` section, as well as the :guilabel:`Recruiter` field, are all
+pre-populated.
+
+Enter the following information on the new applicant form. Note that not all fields listed below may
+be visible. Depending on installed applications and configurations, some fields may not be
+displayed.
+
+.. _recruitment/applicant-details:
+
+Applicant section
+-----------------
+
+- :guilabel:`Subject/Application Name`: this is the only required field. Enter the title for the
+  card. Typically this is the applicant's name and job position being applied to, for example:
+  `John Smith - Experienced Developer`. This field is not visible in the Kanban view, unless the
+  :guilabel:`Applicant's Name` is left blank. If there is no :guilabel:`Applicant's Name`, then the
+  :guilabel:`Subject/Application Name` is what is displayed on the applicant card in the Kanban
+  view.
+- :guilabel:`Applicant's Name`: enter the applicant's name.
+- :guilabel:`Email`: enter the applicant's email address.
+- :guilabel:`Phone`: enter the applicant's phone number.
+- :guilabel:`Mobile`: enter the applicant's mobile number.
+- :guilabel:`LinkedIn Profile`: enter the web address for the applicant's personal profile on
+  LinkedIn.
+- :guilabel:`Degree`: select the applicant's highest level of education earned from the drop-down
+  menu. Options are :guilabel:`Graduate`, :guilabel:`Bachelor Degree`, :guilabel:`Master Degree`, or
+  :guilabel:`Doctoral Degree`. The :guilabel:`Graduate` option indicates graduating the highest
+  level of school before a Bachelor's degree, such as a high school or secondary school diploma,
+  depending on the country.
+- :guilabel:`Interviewers`: using the drop-down menu, select the people who will conduct the
+  interviews. The selected people must have either *recruiter* or *officer* rights configured for
+  the *Recruitment* application to appear in the drop-down list. Refer to the :doc:`Access rights
+  <../../general/users/access_rights>` document for more information.
+- :guilabel:`Recruiter`: select the user responsible for the entire recruitment process for the job
+  position.
+- :guilabel:`Evaluation`: represents a rating for the applicant: one star is :guilabel:`Good`, two
+  stars is :guilabel:`Very Good`, and three stars is :guilabel:`Excellent.`
+- :guilabel:`Source`: using the drop-down menu, select where the applicant learned about the job
+  position. The following options come pre-configured in Odoo: :guilabel:`Search engine`,
+  :guilabel:`Lead Recall`, :guilabel:`Newsletter`, :guilabel:`Facebook`, :guilabel:`Twitter`,
+  :guilabel:`LinkedIn`, :guilabel:`Monster`, :guilabel:`Glassdoor`, and :guilabel:`Craigslist`. To
```

Comment:
There's more to this list (doesn't just end at Craigslist... so you may want to rework this sentence to allude to there being more options available

Reply:
At least that's what's showing up when I click 'Search More...` from the resulting drop-down menu in this field

Reply:
You're seeing extra options put in the runbot- on a blank DB these are the options, with no extra items added to it (same for the next one for the medium) .